src/olympia/users/templates/admin/users/userprofile/change_form.html

```diff
@@ -14,6 +14,12 @@
                  <button formaction="{% url 'admin:users_userprofile_ban' original.pk|admin_urlquote %}"
```

Comment:
you forgot to remove this block still referencing `can_ban`

Reply:
woops good eye, fixed in ab63a486f8627b7308d5cb3c27bafdbb65099372